manifests/init.pp

```diff
@@ -70,11 +63,21 @@
 
   case $facts['os']['family'] {
     'RedHat': {
+      $os_package_require = []
     }
     'Debian': {
+      $os_package_require = [Class['::apt::update']]
     }
     default: {
       fail("Detected osfamily <${::osfamily}>. Only RedHat and Debian are supported.")
     }
   }
+
+  if $manage_repo {
+    include ::sensu::repo
+    $package_require = [Class['::sensu::repo']] + $os_package_require
+  } else {
+    $package_require = undef
```

Comment:
Perhaps we should document which class(es) this `$package_require` variable is meant to be used in. Since it is just specified here and not used, it makes the code harder to understand.

Reply:
Add comment about where `$package_require` is used.